tailwind.config.ts

```diff
@@ -0,0 +1,103 @@
+// See the Tailwind configuration guide for advanced usage
+// https://tailwindcss.com/docs/configuration
+
+const plugin = require("tailwindcss/plugin");
+const fs = require("fs");
+const path = require("path");
+
+module.exports = {
+  content: [
+    "./js/**/*.js",
+    "./js/**/*.ts",
+    "./js/**/*.tsx",
+    "./lib/orbit_web.ex",
+    "./lib/orbit_web/**/*.*ex",
```

Comment:
git didn't notice that this file was moved and edited, rather than deleted and recreated.

The only change in this file aside from formatting was this list of places to look for tailwind classes.